test/fixtures/xruby/exercises/alpha/example.tt

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
```

Comment:
This is a fixture, the folder as an example could be named `foo` or `bar` as traditionally used as examples, even working examples.

`alpha` may suggest there will be a `beta` and `delta`, etc.